std-bits/snowflake/src/main/java/org/enso/snowflake/SnowflakeJDBCUtils.java

```diff
@@ -0,0 +1,39 @@
+package org.enso.snowflake;
+
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class SnowflakeJDBCUtils {
+  private static final DateTimeFormatter dateTimeWithOffsetFormatter =
+      DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSSSSSSSS XXX");
```

Comment:
TODO: this is probably why I have the 'remove T' logic later. I probably should update this to
```suggestion
      DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss.SSSSSSSSS XX");
```
or sth